app/src/features/projects/components/ProjectIUCNForm.tsx

```diff
@@ -68,8 +68,8 @@ export const ProjectIUCNFormYupSchema = yup.object().shape({
 
 export interface IProjectIUCNFormProps {
   classifications: IMultiAutocompleteFieldOption[];
-  subClassifications1: IMultiAutocompleteFieldOption[];
-  subClassifications2: IMultiAutocompleteFieldOption[];
+  subClassifications1: IIUCNSubClassification1Option[];
+  subClassifications2: IIUCNSubClassification2Option[];
```

Comment:
These interfaces were defined in the file above, but were never used.  Just something random I noticed while looking through this code.